shared/src/business/useCases/trialSessions/getTrialSessionsForJudgeInteractor.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  ROLE_PERMISSIONS,
+  isAuthorized,
+} from '../../../authorization/authorizationClientService';
+import { TrialSession } from '../../entities/trialSessions/TrialSession';
+import { UnauthorizedError } from '../../../errors/errors';
+
+/**
+ * getTrialSessionsForJudgeInteractor
+ *
+ * @param {object} applicationContext the application context
+ * @returns {Array<TrialSession>} the trial sessions returned from persistence
+ */
+export const getTrialSessionsForJudgeInteractor = async (
```

Comment:
🔧  What do you think about `getTrialSessionsByPresidingJudgeInteractor`? Just so it's clear that not only authenticated judges might use this endpoint

Reply:
since the interactor requires a judgeId to work, I'm not sure adding Presiding helps me understand the intent any better.

Reply:
Gotcha, I was thinking it was more indicative that the trial sessions being returned were those that the provided judge ID was assigned to. I don't feel super strongly though.